l-homme.csl

```diff
@@ -0,0 +1,221 @@
+      </if>
+      <else>
+        <group delimiter="">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <citation and="text" et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=", ">
+    <sort>
+      <key variable="issued"/>
+    </sort>
+    <layout prefix=" (" suffix=")" delimiter="&#160;; ">
+      <text macro="author-short" suffix=", "/>
+      <text macro="year-date"/>
+      <group>
+        <text macro="loc-cite" prefix="&#160;: "/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography subsequent-author-substitute="" entry-spacing="0">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+      <key variable="page-first"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix="">
+      <group display="block">
+        <text macro="author" suffix=" "/>
+      </group>
+      <group display="left-margin">
+        <text macro="year-date" suffix=" "/>
+        <choose>
+          <if type="thesis">
+            <group suffix=".">
+              <text macro="title" font-style="italic"/>
+            </group>
+            <group delimiter=", " suffix=".">
+              <text variable="genre"/>
+              <text variable="publisher"/>
+              <text variable="publisher-place"/>
+              <group>
+                <text variable="number-of-volumes" suffix="&#160;vol."/>
+                <text variable="number-of-pages" suffix="&#160;p."/>
+              </group>
+            </group>
+          </if>
+          <else-if type="webpage">
+            <group suffix=". ">
+              <text macro="title" font-style="italic"/>
+            </group>
+            <group>
+              <text variable="URL" prefix=", "/>
+              <text prefix=" (" macro="access" suffix=")"/>
+            </group>
+          </else-if>
+          <else-if type="article-journal article-magazine article-newspaper broadcast personal_communication thesis entry-dictionary entry-encyclopedia" match="any">
+            <group suffix=". ">
+              <text macro="title" quotes="true"/>
+            </group>
+            <group>
+              <text variable="container-title" font-style="italic"/>
+              <text variable="volume" prefix=", vol.&#160;"/>
+              <text variable="issue" prefix=", n°&#160;"/>
+              <text macro="pages"/>
+            </group>
+          </else-if>
+          <else-if type="book graphic" match="any">
+            <group suffix=". ">
+              <text macro="title"/>
+            </group>
+            <group delimiter=", " suffix=". ">
+              <text macro="edition"/>
+              <text macro="publisher"/>
+            </group>
+          </else-if>
+          <else-if type="chapter paper-conference" match="any">
+            <text macro="title" quotes="true" suffix=". "/>
+            <group>
```

Comment:
Similarly here: put "in" and "editor" in a group with `delimiter=" "`, set the delimiter for this whole group to `", "` and move pages out of the group. Will make this more robust for missing variable.

Reply:
Yup. Done.